doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -164,132 +164,119 @@ For more information about the ZAP tool, see the official `ZCL Advanced Platform
+      * Configure the tool and manually compile it using the instructions in the official `ZCL Advanced Platform`_ documentation.
 
-        Your currently installed ZAP tool version 2022.12.19 does not match the recommended one: 2022.12.20.
+      Both these methods still require adding the ZAP tool location to the system :envvar:`PATH` environment variables, as detailed in the step 3 above.
 
-     Alternatively, this message can look like the following one:
+..
 
-     .. code-block::
+.. _ug_matter_gs_tools_matter_west_commands:
 
-        No ZAP tool version was found installed on this device.
+Matter west commands
+********************
 
-     In this case, the package download process will start automatically:
+Matter west commands are a set of commands dedicated for Matter-related purposes and integrated with the :ref:`zephyr:west`.
+You can use them to simplify work with the Matter application project.
 
-     .. parsed-literal::
-        :class: highlight
+.. _ug_matter_gs_tools_matter_west_commands_zap_tool:
 
-        Trying to download ZAP tool package matching your system and recommended version.
-        100% [......................................................................] 150136551 / 150136551
-        ZAP tool package was downloaded and extracted in the given location.
+ZAP tool west commands
+======================
 
-     Depending on your operating system, the download will conclude with the following message, with updated *zap_cli_location* and *zap_location* paths, which you will need in the following step:
+To simplify work with the :ref:`ug_matter_gs_tools_zap`, there are two commands implemented within the Matter west commands set:
 
-     .. tabs::
+* **zap-gui**
+* **zap-generate**
 
-        .. group-tab:: Windows
+.. _ug_matter_gs_tools_matter_west_commands_zap_tool_gui:
 
-           .. parsed-literal::
-              :class: highlight
+ZAP gui command
+---------------
 
-              #######################################################################################
-              # Please add the following location(s) to the system PATH:                            #
-              # *zap_location*                                                                      #
-              #######################################################################################
+The ZAP tool GUI is a Node.js application used to configure the data model of a Matter application.
+It allows you to define endpoints, clusters, commands, attributes, and events for a specific application.
+This command installs the appropriate version of the ZAP tool on your host machine and launches the ZAP tool GUI.
 
-        .. group-tab:: Linux
+By default, this command looks for a :file:`.zap` file in the current directory.
+If your console is directed to a different directory than the one shown in the example, you must explicitly specify the path to the :file:`.zap` file as an additional argument.
```

Comment:
```suggestion
You can also specify the path to the :file:`.zap` file explicitly using the `-z` argument.
```
"If your console is directed" is a bit unclear to me so it's better to keep it simple :)

Reply:
I like that simplification.